docs/uma-tokenholders/approved-collateral-currencies.md

```diff
@@ -75,3 +75,4 @@ sidebar_label: Approved Collateral Currencies
 |Total| No of Approved collateral currencies | [66](https://docs.umaproject.org/uma-tokenholders/approved-collateral-currencies)
```

Comment:
This hard coded 66 is going to have a hard time staying up to date

Reply:
It's updated every week :)